CMakeLists.txt

```
@@ -134,6 +135,10 @@ else()
   message(STATUS "Not using HIP")
 endif(RESOLVE_USE_HIP)
 
+if (RESOLVE_USE_ASAN)
+  set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -fsanitize=address -fno-omit-frame-pointer")
+  set(CMAKE_EXE_LINKER_FLAGS "${CMAKE_EXE_LINKER_FLAGS} -fsanitize=address")
+endif()
```

My suggestion is to minimize the use of literal strings:
 ```suggestion
if (RESOLVE_USE_LLVM_ADDRESS_SANITIZER)
  set(CMAKE_CXX_FLAGS ${CMAKE_CXX_FLAGS} "-fsanitize=address -fno-omit-frame-pointer")
  set(CMAKE_EXE_LINKER_FLAGS ${CMAKE_EXE_LINKER_FLAGS} "-fsanitize=address")
endif()
```
I am guessing that @cameronrutherford would also suggest using CMake lists instead of sets here.